internal/db/db.go

```diff
@@ -37,14 +37,15 @@ type CloneEntry struct {
 
 type BlueprintEntry struct {
 	Id          uuid.UUID
+	VersionId   uuid.UUID
```

Comment:
This column is NULLable, therefore you have to use pointer here:

https://github.com/osbuild/image-builder/blob/main/internal/db/migrations-tern/009_create_table_blueprint_versions.sql#L18C1-L18C117

Scanning a NULL value into non-pointer will lead to panic.

Reply:
I don't think the version can ever be nil :thinking: or can it?

Reply:
Oh, you're pointing to a Compose table, but this is a BlueprintEntry.
We do not  scan for the Compose blueprint_version_id column here, if I read correctly.